src/CampaignAdmin.php

```diff
@@ -313,12 +312,15 @@ protected function getChangeSetResource(ChangeSet $changeSet, $sync = false)
                 }
             }
 
-        // An unexpected data exception means that the database is corrupt
+            // An unexpected data exception means that the database is corrupt
         } catch (UnexpectedDataException $e) {
             $hal['PublishedLabel'] = '-';
             $hal['Details'] = 'Corrupt database! ' . $e->getMessage();
             $hal['canPublish'] = false;
         }
+
+        $this->extend('updateChangeSetResources', $hal, $changeSet);
```

Comment:
Maybe this should be `updateChangeSetResource` given the function name is `getChangeSetResource`?

Reply:
@dhensby I've updated the name of the method.